client/components/single-lesson.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router-dom'
+import { fetchLesson } from '../store'
+
+class Lesson extends Component {
+  componentDidMount() {
+    this.props.getLesson(+this.props.match.params.lessonId);
+  }
+
+  render() {
+    const lesson = this.props.lesson;
+    return (
+      <div>
+        <ul>
```

Comment:
enclose <li>s directly with a <ul>, remove interior <div>, add key back to containing <div>